src/applications/vaos/components/layout/InPersonLayout.jsx

```diff
@@ -135,6 +140,16 @@ export default function InPersonLayout({ data: appointment }) {
         <br />
         <span>Other details: {`${otherDetails || 'Not available'}`}</span>
       </Section>
+      {featureMedReviewInstructions && (
```

Comment:
From what I can tell, In-person and COVID vaccine appointments both use the `InPersonLayout` so this change will cover both of these scenarios/ACs. Let me know if this isn't the case though.